health-services/plan-service/src/main/java/digit/util/MdmsV2Util.java

```diff
@@ -0,0 +1,74 @@
+package digit.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import digit.config.Configuration;
+import lombok.extern.slf4j.Slf4j;
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.tracer.model.CustomException;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.client.RestTemplate;
+import digit.web.models.mdmsV2.*;
+
+import java.util.*;
+
+import static digit.config.ServiceConstants.*;
+
+@Slf4j
+@Component
+public class MdmsV2Util {
+
+    private RestTemplate restTemplate;
+
+    private ObjectMapper objectMapper;
+
+    private Configuration configs;
+
+    public MdmsV2Util(RestTemplate restTemplate, ObjectMapper objectMapper, Configuration configs)
+    {
+        this.restTemplate = restTemplate;
+        this.objectMapper = objectMapper;
+        this.configs = configs;
+    }
+
+    public List<Mdms> fetchMdmsV2Data(RequestInfo requestInfo, String tenantId, String schemaCode)
+    {
+        StringBuilder uri = getMdmsV2Uri();
+        MdmsCriteriaReqV2 mdmsCriteriaReqV2 = getMdmsV2Request(requestInfo, tenantId, schemaCode);
+        MdmsResponseV2 mdmsResponseV2 = null;
+        try{
```

Comment:
reformat this piece of code - after try there should be a space then you should put a bracket